pkg/snmp/traps/formatter.go

```diff
@@ -48,21 +49,26 @@ func NewJSONFormatter(oidResolver OIDResolver) (JSONFormatter, error) {
 
 // FormatPacket converts a raw SNMP trap packet to a FormattedSnmpPacket containing the JSON data and the tags to attach
 func (f JSONFormatter) FormatPacket(packet *SnmpPacket) ([]byte, error) {
-	var formattedData map[string]interface{}
+	payload := make(map[string]interface{})
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Could help to add an example payload to the above comment